vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/permission_rules/minimum_wait_time_frequency_cap.cc

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/permission_rules/minimum_wait_time_frequency_cap.h"
+#include "bat/ads/internal/frequency_capping/frequency_capping.h"
+#include "bat/ads/ads_client.h"
+#include "bat/ads/internal/time.h"
+#include "bat/ads/internal/client.h"
+#include "bat/ads/internal/ads_impl.h"
+
+#include "bat/ads/ad_info.h"
+
+namespace ads {
+
+bool MinimumWaitTimeFrequencyCap::IsAllowed() {
+  if (ads_->IsMobile()) {
+    return true;
+  }
+
+  auto history = frequency_capping_->GetAdsHistory();
+
+  auto respects_hour_limit = AreAdsPerHourBelowAllowedThreshold(history);
+  if (!respects_hour_limit) {
+    std::ostringstream string_stream;
+    string_stream << "IsAllowed: respects_hour_limit: " << respects_hour_limit;
+    reason_for_exclusion_ = string_stream.str();
+    return false;
+  }
+  auto respects_minimum_wait_time = AreAdsAllowedAfterMinimumWaitTime(history);
+  if (!respects_minimum_wait_time) {
+    std::ostringstream string_stream;
+    string_stream << "IsAllowed: respects_minimum_wait_time:" <<
```

Comment:
Please change log message to "The minimum wait time has not passed before an ad can be shown". We do not need the flag to be output

Reply:
Done